brouter-core/src/main/java/btools/router/StdPath.java

```diff
@@ -139,7 +143,45 @@ else if ( ehbu < 0 )
     }
 
     sectionCost += dist * costfactor + 0.5f;
-      
+
+    if (rc.bikeMode || rc.footMode) {
+      // TODO: Should be moved to profile
+      // Gravitational constant, g
+      double GRAVITY = 9.81;  // in meters per second^(-2)
+      // Total mass of bike + biker + luggages, m
+      double MASS = 90;  // in kilograms
+      // Speed when we start braking and never go beyond
+      double MAX_SPEED = 45. * 3.6;  // in meters per second
+      // Equivalent surface for wind, S * C_x, F = -1/2 * S * C_x * v^2 = - S_C_x * v^2
+      double S_C_x = 0.5 * 0.45;  // in meters^2
+      // The resistance of the road, F = - m * g * C_r
+      double C_r = 0.01;
+      // Biker power
+      double BIKER_POWER = 100;  // in W
+
+      // Uphill angle
+      double alpha = Math.atan2(delta_h, distance);
+
+      // Travel speed
+      double speed = Double.NaN;
+      if (rc.footMode) { // TODO: || tags['way'].search('bicycle=dismount') !== -1) {
```

Comment:
Is there an easy way to access the `bicycle=dismount` tag at this point? Thanks!